nyxtools/_version.py

```diff
@@ -190,27 +196,34 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
         # between branches and tags. By ignoring refnames without digits, we
         # filter out many common branch names like "release" and
         # "stabilization", as well as "HEAD" and "master".
-        tags = set([r for r in refs if re.search(r'\d', r)])
+        tags = set([r for r in refs if re.search(r"\d", r)])
         if verbose:
             print("discarding '%s', no digits" % ",".join(refs - tags))
     if verbose:
         print("likely tags: %s" % ",".join(sorted(tags)))
     for ref in sorted(tags):
         # sorting will prefer e.g. "2.0" over "2.0rc1"
         if ref.startswith(tag_prefix):
-            r = ref[len(tag_prefix):]
+            r = ref[len(tag_prefix) :]
```

Comment:
As much as I hate this, I suppose it must be suggested by black/isort/flake?

Reply:
I did not see it in my latest commit https://github.com/NSLS-II-NYX/nyxtools/pull/10/commits/b96745f60da70ad23ef71d1a422a38a002555269, but probably was addressed by the earlier linting. And we usually use this `_version.py` file as is coming from https://github.com/NSLS-II/scientific-python-cookiecutter, so I would not worry about it too much. In some projects, we disable checking against `versioneer.py` and this file in the library.